src/components/20-molecules/policy-features/index.scss

```diff
@@ -0,0 +1,48 @@
+.m-policy-features {
+  padding: 24px 15px;
+
+  @include breakpoint($mediaquery-md-up) {
+    padding: 48px 90px;
+  }
+
+  &__title {
+    text-align: center;
+    margin: 0;
+
+    @include typo-title(large-1, secondary);
+
+    @include breakpoint($mediaquery-md) {
+      @include typo-title(huge, secondary);
+    }
+
+    @include breakpoint($mediaquery-lg-up) {
+      @include typo-title(huge-2, secondary);
+    }
+  }
+
+  &__slot {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+}
+
```

Comment:
I would recommend to create a mini mixing that living in that file. Because the styleing of the variants are redundant and simple.